unet/test_train.py

```diff
@@ -0,0 +1,10 @@
+from model import get_unet
+from train import train_unet
+
+
+img_size = (256, 256)
+
+model = get_unet(img_size, 13)
+model = train_unet(model, 50, 32, img_size, test_size=0.25, dataset_folder="./dataset")
+
+model.save("./unet_model")
```

Comment:
In case we save mulitple models, do we want to save the models with a unique name by using the date? something like this?

Reply:
If you want to code up a CLI that makes use of the training code be my guest. I wanted to get something together really quick to get stuff combined and ready for quick usage.